documentation/src/main/asciidoc/user_guide/streams.adoc

```diff
@@ -534,5 +539,41 @@ public class WordFrequencyExample {
 
 This way you can still utilize all of the cores locally when calculating the most frequent element.
 
+*Remove specific entries*
+
+Distributed streams can also be used as a way to modify data where it lives.
+For example you may want to remove all entries in your cache that contain
+a specific word.
+
+[source,java]
+----
+public class RemoveBadWords {
+   public static void main(String[] args) {
+      // Lines removed
+      String word = ..
+
+      c1.entrySet().parallelStream()
```

Comment:
I have added a paragraph to the Parallel Computation section detailing this more.